src/Planet/Planet.jl

```diff
@@ -127,6 +132,16 @@ function year_anom end
 function orbit_semimaj end
 """ Total solar irradiance (W/m``^2``) """
 function TSI end
+""" J2000 epoch """
+function epoch end
+""" mean anomaly at the epoch (radians) """
+function M_epoch end
+""" orbital obliquity at the epoch (radians) """
+function γ_epoch end
+""" longitude of perihelion at the epoch (radians) """
```

Comment:
```suggestion
""" longitude of perihelion at the epoch (radians), measured relative to vernal equinox (i.e., the longitude of perihelion is the angle subtended at the Sun by the orbital arc from vernal equinox to perihelion)."""
```
(Good to be explicit here, because there are various definitions of the longitude of perihelion in use.)